src/pages/ConnectWallet/ConnectWallet.tsx

```diff
@@ -104,98 +102,107 @@ export const ConnectWallet = () => {
-        py={3}
-        px={4}
-        bottom={0}
-        alignItems={'center'}
+        backgroundImage={AuroraBg}
+        backgroundSize='cover'
+        backgroundPosition='bottom center'
+        width='100vw'
+        alignItems='center'
+        justifyContent='center'
       >
-        <DarkMode>
-          <Flex width='full' alignItems='center' justifyContent='center' gap={8}>
-            <Link href='/#/legal/terms-of-service'>
-              <Text color='gray.500' translation='common.terms' />
-            </Link>
-            <Link href='/#/legal/privacy-policy'>
-              <Text color='gray.500' translation='common.privacy' />
-            </Link>
-          </Flex>
-        </DarkMode>
-      </Flex>
-      <Center
-        flexDir='column'
-        height='100vh'
-        backgroundImage={colors.altBg}
-        px={6}
-        _after={{
-          position: 'absolute',
-          content: '""',
-          left: 0,
-          top: 0,
-          width: '100vw',
-          height: '100vh',
-          backgroundImage: `url(${isFirefox ? OrbsStatic : Orbs})`,
-          backgroundSize: 'cover',
-          backgroundPosition: 'center center',
-        }}
-      >
-        <Circle size='100px' mb={6}>
-          <FoxIcon boxSize='100%' color='white' />
-        </Circle>
-        <Flex flexDir='row' textAlign='center' fontSize={{ base: '6xl', lg: '8xl' }} mb={6}>
-          <RawText color='white' fontWeight='medium' lineHeight='1'>
-            {translate('connectWalletPage.exploreThe')}{' '}
-            <RawText color='blue.500' fontWeight='bold' as='span'>
-              {translate('connectWalletPage.defiUniverse')}
-            </RawText>
-          </RawText>
-        </Flex>
-        <Text
-          color='gray.500'
-          fontSize='lg'
-          mb={12}
-          textAlign='center'
-          translation='connectWalletPage.body'
-        />
-        <Stack
-          alignItems='center'
-          spacing={{ base: 2, md: 8 }}
-          mx='auto'
-          direction={{ base: 'column', md: 'row' }}
-        >
-          <Button
-            size='lg'
-            zIndex={1}
-            colorScheme='blue'
-            onClick={() => dispatch({ type: WalletActions.SET_WALLET_MODAL, payload: true })}
-            data-test='connect-wallet-button'
-          >
-            <Text translation='connectWalletPage.cta' />
-          </Button>
-          <Flex alignItems='center' justifyContent='center'>
+        <Flex flexDir='column' zIndex={4} width='full'>
+          <Center flexDir='column' height='100vh' px={6}>
+            <Circle size='100px' mb={6}>
+              <FoxIcon boxSize='100%' color='white' />
+            </Circle>
+            <Flex
+              flexDir='row'
+              textAlign='center'
+              letterSpacing='-4px'
+              fontSize={{ base: '6xl', lg: '8xl' }}
+              mb={6}
+            >
+              <RawText color='whiteAlpha.400' fontWeight='medium' lineHeight='1'>
```

Comment:
The contrast here is a _little_ low.

<img width="619" alt="Screen Shot 2022-06-23 at 10 52 04 am" src="https://user-images.githubusercontent.com/97164662/175352936-52c90cd2-d7a5-489d-9907-69e33924cad8.png">